sdks/java/io/xml/pom.xml

```diff
@@ -101,7 +101,7 @@
     <dependency>
       <groupId>org.hamcrest</groupId>
       <artifactId>hamcrest-core</artifactId>
-      <version>1.3</version>
+      <scope>test</scope>
```

Comment:
If defined in `dependencyManagement` in the parent they should: https://gist.github.com/aviemzur/2a4dc78b542644ea49601f8ff3068b06